jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.j3md

```diff
@@ -132,6 +132,22 @@ MaterialDef PBR Lighting {
         Boolean UseVertexColor
 
         Boolean BackfaceShadows : false
+
+        Boolean UseVertexColorsAsSunIntensity
+        Float StaticSunIntensity
+        Boolean BrightenIndoorShadows
+    
+        Int DebugValuesMode
+            // debugs the final value of the selected layer as a color output:         
+            // Layers:
+            //   0 - albedo (un-shaded)
+            //   1 - normals
+            //   2 - roughness
+            //   3 - metallic
+            //   4 - ao
+            //   5  - emissive
```

Comment:
Just a few improvements: could you remove the extra space between the 5 and the - to align the list of values and change `(un-shaded)` -> `(unshaded)` ?